packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.types.ts

```diff
@@ -68,10 +56,11 @@ export interface IFocusTrapZoneProps extends React.HTMLAttributes<HTMLDivElement
   disableFirstFocus?: boolean;
 
   /**
-   * If the FocusTrapZone is declared focusable (data-is-focusable=true) and it
-   * receives focus directly, or if .focus() is called, it will pass on
-   * focus to a child element.  This prop determines how that element is selected.
-   * @default FocusTransferRule.firstFocusable
+   * Specifies the algorithm used to determine which descendant element to focus when the FocusTrapZone is focused.
+   * If false, the first focusable descendant, filtered by the firstFocusableSelector property if present, is chosen.
+   * If true, the element that was focused when the Trap Zone last had a focused descendant is chosen.
+   * If it has never had a focused descendant before, behavior falls back to the first focused descendant.
+   * @default false
```

Comment:
I was following the convention of other bool props that default to false: isClickableOutsideFocusTrap, ignoreExternalFocusing and disableFirstFocus.  It's purely for docgen purposes.

Reply:
Those are used in COmponentDidMount and they are actually set to the default values. Ideally if they are going to set a default value they should have used defaultProps. I think it's fine to not put a default value, because it's actually just undefined. You could say `If false or undefined` if you wanted to document it

Also, this is optional it's nice to put that at the beginning of the jsdoc block